Buildings/Fluid/AirFilters/BaseClasses/Data/Generic.mo

```diff
@@ -0,0 +1,49 @@
+within Buildings.Fluid.AirFilters.BaseClasses.Data;
+record Generic "Generic data record for air filters"
+  extends Modelica.Icons.Record;
+  parameter Real mCon_nominal(
+    final unit = "kg")
+    "Maximum mass of the contaminant can be captured by the filter";
```

Comment:
`Maximum mass of the contaminant that can be ...`

Reply:
Fix the typo.